content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -395,3 +395,71 @@ See the following resources:
 - [Transaction isolation levels in MySQL](https://dev.mysql.com/doc/refman/8.0/en/innodb-transaction-isolation-levels.html)
 
 CockroachDB and SQLite only support the `Serializable` isolation level.
+
+### Transaction timing issues
+
+<Admonition type="info">
+
+- The solution in this section does not apply to MongoDB, because MongoDB does not support [isolation levels](https://www.prisma.io/dataguide/intro/database-glossary#isolation-levels).
+- The timing issues discussed in this section do not apply to CockroachDB and SQLite, because these databases only support the highest `Serializable` isolation level.
+
+</Admonition>
+
+When two or more transactions run concurrently in certain [isolation levels](https://www.prisma.io/dataguide/intro/database-glossary#isolation-levels), timing issues can cause conflicts, such as the violation of unique constraints. For example, consider the following sequence of events where Transaction A and Tranasaction B both attempt to execute a `deleteMany` and a `createMany` operation:
+
+1. Transaction A: `deleteMany` operation removes a set of rows.
+1. Transaction B: `deleteMany` operation removes a set of rows.
```

Comment:
Are these two steps even necessary for the problem to trigger?

Reply:
You might be right, but I think there was originally a need for these lines. I'll go back to the original notes and see if I've chopped something important out in one of the revisions. If not, I'll remove these two steps.

Reply:
I think you're right - I've removed these two steps